admin.py

```diff
@@ -0,0 +1,81 @@
+from flask import Flask, render_template, jsonify, request
+from flask_cors import CORS
+from flask_pymongo import PyMongo
+from pymongo import MongoClient
+
+import db
+import json
+import datetime
+from bson import json_util, ObjectId
+from lib import errors
+from lib.utils import clean_json, error, expect_json, success
+
+app = Flask(__name__)
+app.config["SECRET_KEY"] = "ttmadmin"
+CORS(app)
+mdb = MongoClient(db.MONGO_URL).db
+
+@app.route("/auth", methods=["POST"])
+@expect_json(password=str)
+def admin_auth(body):
+    if(body['password'] == "ttmadmin"):
+        return success("Authorized")
+    return error(403, "Incorrect password")
+
+@app.route("/reports", methods=["GET"])
+def get_reports():
+    reports = mdb.reports.find()
+    data=[]
+    for x in reports:
+        x['_id'] = str(x['_id'])
+        data.append(x)
+    jsonData=json.dumps(data)
+    return jsonData

```

Comment:
Make sure to return `success(jsonData)` rather than just the data (same for other endpoints below)